docs/sphinx/source/changelog/v2.0.0.rst

```diff
@@ -10,6 +10,8 @@ API Changes
   right-labeled energy with :py:func:`~rdtools.normalization.energy_from_power`
   before being used with these normalization functions (:pull:`105`, :pull:`108`).
 * Remove ``low_power_cutoff`` parameter in :py:func:`~rdtools.filtering.clip_filter` (:issue:`84`).
+* :py:func:`~rdtools.normalization.delta_index` and :py:func:`~rdtools.normalization.check_series_frequency`
+  made private functions. (:pull:`222`)
```

Comment:
Oh, didn't realize these functions weren't new in 2.0.  I think the Right Thing To Do would be to deprecate them and remove in 3.0, but maybe we don't think anybody would care if they get removed without deprecation?

Reply:
Ah good point. Yes, let's do The Right Thing.

Reply:
Actually this causes the functions that rely on them to emit deprecation warnings. If there isn't an easy way around that, I don't think it's a big deal to make them private. I'd be surprised if anyone is using them, they were already omitted from `__init__.py`

Reply:
We could catch them where they are called (https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings) But I'm not sure it's worth the maintenance. I could go either way, preference?

Reply:
I think the `@deprecated` decorator can handle that.  Make the function private, switch any callers to using the new private name, and make a deprecated public version that just wraps the private one (calling the decorator directly instead of using it as a decorator, it's a little weird but it works):

```python
from rdtools._deprecation import deprecated

def _delta_index(...):
    ....

delta_index = deprecated('2.0.0')(_delta_index)
```
So users calling `delta_index` will get the deprecated warning, but our functions that call `_delta_index` won't. I didn't test it but in theory it should work...